src/test/test.ts

```diff
@@ -1,11 +1,80 @@
 import * as index from "../index"
 import * as assert from "assert"
+import * as path from "path"
 
-jest.setTimeout(10000);
+jest.setTimeout(10000)
 
 describe("index", () => {
   it("simple", async () => {
     const diff = new index.OpenApiDiff({})
-    const result = await diff.compare("src/test/simple/same.json", "src/test/simple/same.json")
+    const file = "src/test/simple/same.json"
+    const resultStr = await diff.compare(file, file)
+    const result = JSON.parse(resultStr)
+    const filePath = path.resolve(file).split("\\").join("/")
+    const expected = [
+      {
+        code: "NoVersionChange",
+        docurl: "https://github.com/Azure/openapi-diff/tree/master/docs/rules/1001.md",
+        id: "1001",
+        message: "The versions have not changed.",
+        mode: "Update",
+        new: {
+          ref: `file:///${filePath}#`,
+          path: "",
+          location: `file:///${filePath}:1:1`
+        },
+        old: {
+          ref: `file:///${filePath}#`,
+          path: "",
+          location: `file:///${filePath}:1:1`
+        },
+        type: "Info"
+      }
+    ]
+    assert.deepStrictEqual(result, expected)
+  })
+  it("some-changes", async () => {
+    const diff = new index.OpenApiDiff({})
+    const oldFile = "src/test/some-changes/old.json"
+    const newFile = "src/test/some-changes/new.json"
+    const resultStr = await diff.compare(oldFile, newFile)
```

Comment:
if you wanted you could also add a test swapping the parameters, to check on deletion of path and location of "new" as non-existent - which should also flag a "breaking" change. I can't remember how the "breaking ones are being flagged.